ini_python_multiline_test.go

```diff
@@ -0,0 +1,50 @@
+package ini_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type testData struct {
+	Value1 string `ini:"value1"`
+	Value2 string `ini:"value2"`
+}
+
+func TestMultiline(t *testing.T) {
+	Convey("Parse Python-style multiline values", t, func() {
+
+		// enable Python-style multiline values
+		opts := ini.LoadOptions{
+			AllowPythonMultilineValues: true,
+		}
+
+		// load test data
+		path := filepath.Join("testdata", "multiline.ini")
+		f, err := ini.LoadSources(opts, path)
```

Comment:
No need to define `opts` since it is only used once, and to match style in every other places of tests.

```go
f, err := ini.LoadSources(ini.LoadOptions{
	AllowPythonMultilineValues: true,
}, path)
```

Reply:
OK, adopted